### ISSUE TITLE
Notification - Email Update takes Student to Requestor Profile Page

### ISSUE THREAD

**Issue by jwhipple26** [WeStudee/WeStudee#77]
To test the notification process, then I created a new account and send a study request.  (To avoid using Joel for all IDs, I used Jorge.)

![screenshot 2015-05-10 09 44 45](https://cloud.githubusercontent.com/assets/10063859/7554505/41c298fa-f6f9-11e4-9f80-156a757da64d.png)

The email notification is good, except the URL is not hyperlinked.  I think this is inconsistent because the link worked when I replicated the test.  I copy/pasted the URL into a new browser window and was taken to the following page:

![screenshot 2015-05-10 09 45 31](https://cloud.githubusercontent.com/assets/10063859/7554514/7df64c86-f6f9-11e4-9016-666ccb3609a6.png)

To summarize:  Jorge requested to study with Joel.  Joel received an email notification.  Joel clicked on the link to review Jorge and was taken to Jorge's profile page, which is open to edit.
This test occurred in Chrome on a Mac.

**Comment by jwhipple26** [comment]
I replicated again (Joel to request study with Jorge) and the link in the email went to the correct accept/deny page. (Again, the URL was not hyperlinked.)

In this test, then the request was sent from Safari.  The email was received via Chrome and the link was opened in Chrome.  I have no idea if the browser would impact these links/destination, but I'm guessing it more of a stability thing that is causing the inconsistency.

**Comment by yutarochan** [comment]
Email links whether they work or not is dependent on many factors such as what email client you are using, browser and others. The best I can do is instruct them to copy and paste the URL onto the address bar. I will look into this a little bit more during redesigning of the interface.
